api/__tests__/app.test.js

```diff
@@ -89,3 +89,51 @@ describe('GET api/articles', () => {
 
 
 
+describe.only('GET api/articles/:article_id', () => {
+    test('should return a 200 status', () => {
+        return supertest(app).get('/api/articles/1').expect(200)
+    
+    })
+    test('should return an object representing the correct article', () => {
+        return supertest(app).get('/api/articles/1').expect(200)
+        .then(({body}) => {
+            const bodySize = Object.keys({body}).length
+            expect(bodySize).toEqual(1); 
+            console.log(body.articles)
+            const size = Object.keys(body.articles).length;
+            expect(size).toEqual(8)
+
+                    expect(body.articles).toHaveProperty('author');
```

Comment:
refactor suggestion: your test is made more robust if you check the data type expected on each key, rather then whether the key exists (expect.any(String), for example)